src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,49 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+
+    private static final int MAX_SIZE = 10;
+
+    private K[] keyArray;
+    private V[] valueArray;
+    private int size = 0;
+
+    public StorageImpl() {
+        this.keyArray = (K[]) new Object[MAX_SIZE];
+        this.valueArray = (V[]) new Object[MAX_SIZE];
```

Comment:
```suggestion
        keyArray = (K[]) new Object[MAX_SIZE];
        valueArray = (V[]) new Object[MAX_SIZE];
```
we should use `this` if the name of input parameters are the same as fields in the class